internal/handlers/registry/registry.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package registry
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"github.com/FerretDB/FerretDB/internal/handlers"
+)
+
+// NewHandler represents a function that constructs a new handler.
+type NewHandler func(opts *NewHandlerOpts) (handlers.Interface, error)
+
+// NewHandlerOpts represents common configuration for constructing handlers.
+type NewHandlerOpts struct {
+	PostgresURL string
+	TigrisURL   string
```

Comment:
PostgresURL and TigrisURL are not common – they are handler-specific

Reply:
> PostgresURL and TigrisURL are not common – they are handler-specific

removed the word "common" from the comment.